src/azul/service/responseobjects/step_function_helper.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from azul.deployment import aws
+
+
+class StepFunctionHelper:
```

Comment:
This class isn't really necessary because it has no state or class attributes.  These methods can just be functions in the file.  Is there any preference on which style to use?  I don't have any strong opinions in either direction.

Reply:
Class is fine. With a module, you'd have to import each function individually or use `*` import which is considered harmful.